### PR TITLE
CI: Move to GitHub Actions

### LINKED ISSUE #4: Port GitLab CI script to Github Actions
It would be welcome to the project to have a running test suite on every commit, especially to prevent cases such as #1 where you'd be relying on manual testing.

### PR DESCRIPTION
**Pull request by lexisother**
Fixes #4

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          - 5432:5432
+        # needed because the postgres container does not provide a healthcheck
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Show Python version info
+        run: python -V
+
```
Comment (dolfies) [inline .github/workflows/ci.yml:33]:
```suggestion
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      - name: Install dependencies
+        run: time apk --update add --no-cache build-base gcc libgcc libffi-dev openssl-dev git postgresql-client jpeg-dev openjpeg-dev zlib-dev freetype-dev lcms2-dev freetype-dev gifsicle rust cargo
```
Comment (dolfies) [inline .github/workflows/ci.yml:32]:
```suggestion
        run: apt install build-essential gcc libgcc-s1 libffi-dev libssl-dev git postgresql-client libjpeg-dev libopenjp2-7-dev zlib1g-dev libfreetype-dev liblcms2-dev gifsicle
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+      - name: Upgrade pip
+        run: time pip3 install --upgrade pip
+
```
Comment (dolfies) [inline .github/workflows/ci.yml:36]:
```suggestion
```

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+      - name: Install Python dependencies
+        run: time pip3 install wheel tox
```
Comment (dolfies) [inline .github/workflows/ci.yml:38]:
```suggestion
        run: time pip3 install -U pip wheel tox
```

File: .github/workflows/ci.yml (hunk 2 above)
Comment (lexisother) [reply .github/workflows/ci.yml:32]: why are you telling me to use apt if we're running in alpine

Comment (dolfies) [reply .github/workflows/ci.yml:32]: eyes go brrr

**Review by dolfies — APPROVED** [review]

**Pull request merged**